src/adv.py

```diff
@@ -49,3 +52,13 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+while True:
+    directions = ['n', 's', 'e', 'w']
+    cmd = input("Enter a command ")
+    if (cmd == 'q'):
+        print("Quitting game...")
+        break
+    elif (cmd in directions):
```

Comment:
nice thinking to move this to a different class so your while loop doesn't become too big!